packages/url_launcher/url_launcher/README.md

```diff
@@ -8,6 +8,23 @@ iOS, Android, web, Windows, macOS, and Linux.
 ## Usage
 To use this plugin, add `url_launcher` as a [dependency in your pubspec.yaml file](https://flutter.dev/platform-plugins/).
 
+## Installation
+
+### iOS 
+
+When targeting iOS 9.0+, new default from Flutter 1.22+.
+Add the following keys to your _Info.plist_ file, located in `<project root>/ios/Runner/Info.plist`:
+
+```
+<key>LSApplicationQueriesSchemes</key>
+<array>
```

Comment:
Does this cover other [Schemes](https://developer.apple.com/library/archive/featuredarticles/iPhoneURLScheme_Reference/Introduction/Introduction.html) like `mailto`, `tel` and `sms`, or we must add new items for this property?

Reply:
Those schemes are supported without adding this so no new items are required